apps/fz_http/test/support/fixtures/notifications_fixtures.ex

```diff
@@ -0,0 +1,18 @@
+defmodule FzHttp.NotificationsFixtures do
+  @moduledoc """
+  This module defines test helpers for creating notifications.
+  """
+
+  @doc """
+  Generate a notification.
+  """
+  def notification_fixture(attrs \\ %{}) do
+    %{
+      type: :error,
+      user: "test@localhost",
+      message: "Notification test text",
+      timestamp: DateTime.utc_now()
+    }
+    |> Map.merge(attrs)
+  end
```

Comment:
```suggestion
  def notification_fixture(attrs \\ []) do
    Enum.into(attrs, %{
      type: :error,
      user: "test@localhost",
      message: "Notification test text",
      timestamp: DateTime.utc_now()
    })
  end
```

This way you can pass in either keyword lists or maps. e.g. `notification_fixture(type: :info)`